doc/requirements.txt

```
@@ -0,0 +1,6 @@
+sphinx==4.5.0
+pillow==9.1.1
+sphinx-gallery==0.10.1
+m2r2
+sphinx-automodapi==0.13
+sphinx-copybutton

```

Let's alphabetize this and add pins to the missing dependencies!
```suggestion
pillow==9.1.1
m2r2
sphinx==4.5.0
sphinx-automodapi==0.13
sphinx-copybutton
sphinx-gallery==0.10.1
```

As a side note, we recently started using Poetry for some internal projects and are quite happy with its dependency management system. It could be worth introducing it to some of Xanadu's OSS projects as well!

---

Yes! Good idea.